### ISSUE TITLE
Extra class at last step

### ISSUE THREAD

**Issue by Martinomagnifico** [linkedin/hopscotch#211]
Can we have an extra class on the last step?
I use visual cues (arrows) in my buttons, but I would like to not see it in the "Ready" button on the last step. I can put this in my last step JSON: 

                        showNextButton: false,
                        showCTAButton: true,
                        ctaLabel: "Ready"

and style the CTA button to not show the arrow, but I prefer not to do it that way. First, the 'last' button is not a Call To Action, and second, then I have to include the above config in every tour I have on the last step. I think it should be done automatically, either by an added class on the whole hopscotch element, or just on the last next button.

**Comment by Martinomagnifico** [comment]
And to continue the last post: Clicking the CTA button doesn't close the bubble.

**Comment by zimmi88** [comment]
Yeah, I'm afraid the CTA button doesn't automatically move to the next step (or end the tour). You can add this behavior through that step's callback, but it's not assumed.

I'll mark this as a feature enhancement we can consider for the default template. In the meantime, some workarounds that might work for your use case...
- You can use the `onShow` callback to add a custom class. The markup for the bubble should exist by the time the `onShow` callback is invoked.
- The markup for hopscotch bubbles are rendered using JavaScript templates. Usually, if you need particular markup for a given tour, we recommend creating your own template and registering it with Hopscotch. To be fair, in this case that might be overkill. But, you can grab the latest stable release of Hopscotch, change the default template's markup, then use Grunt to rebuild a custom version of Hopscotch to suit your needs. See https://github.com/linkedin/hopscotch/blob/master/src/tl/README.md and https://github.com/linkedin/hopscotch/blob/master/README.md for details.

I hope this helps!
